### PR TITLE
Here's Johnny!

### PR DESCRIPTION
**Pull request by BillyGalbreath**

### CODE REVIEW

File: Spigot-Server-Patches/0390-Here-s-Johnny.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+diff --git a/src/main/java/net/minecraft/server/EntityVindicator.java b/src/main/java/net/minecraft/server/EntityVindicator.java
+index 597878e2c..b36d573af 100644
+--- a/src/main/java/net/minecraft/server/EntityVindicator.java
++++ b/src/main/java/net/minecraft/server/EntityVindicator.java
+@@ -48,6 +48,16 @@ public class EntityVindicator extends EntityIllagerAbstract {
+         this.a(1, flag);
+     }
+ 
++    // Paper start
++    public boolean isJohnny() {
```
Comment (aikar) [inline Spigot-Server-Patches/0390-Here-s-Johnny.patch]: obf helpers go on same line as the variable

**Review by aikar — CHANGES_REQUESTED** [review]

File: Spigot-Server-Patches/0390-Here-s-Johnny.patch [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
++        return b;
++    }
++
++    public void setJohnny(boolean johnny) {
++        b = johnny;
++    }
++    // paper end
```
Comment (kashike) [inline Spigot-Server-Patches/0390-Here-s-Johnny.patch]: `Paper`!

**Review by kashike — DISMISSED** [review]

**Review by Spottedleaf — APPROVED** [review]

**Comment by kashike** [comment]
hi

**Comment by kashike** [comment]
billy

**Review by aikar — APPROVED** [review]

**Pull request merged**